src/dataverk/datapackage.py

```diff
@@ -96,46 +94,27 @@ def path(self):
     def url(self):
         return self._datapackage_metadata.get("url")
 
-    def add_resource(self, resource: Any, resource_type: str, resource_name: str,
+    def add_resource(self, resource: Any, resource_type: str, resource_name: str = "",
                      resource_description: str = "", spec: dict = None):
         """
-        :param resource:
-        :param resource_type:
-        :param resource_name:
-        :param resource_description:
-        :param spec:
-        :return:
+        :param resource: Resource to be added to Datapackage
+        :param resource_type: Type of resource
+        :param resource_name: Name of resource
+        :param resource_description: Description of resource
+        :param spec: Resource specification e.g format, compress, etc
+        :return: None
```

Comment:
Nice! 
Kanskje utvide med å
- legge til en toppbeskrivelse i doc
- legge til hvilke type ressurser som går an i 'resource type'? (["df", "pdf", "remote"]) som du har gjort for spec og i tillegg forklare forskjellen mellom dem hvis ikke det blir forklart over
- legge til default=default i slutten av beskrivelsene til parameterne som har default 
- legge til type variabel hvis ikke beskrivelsen forklarer det godt nok.

eks: 
:param: resource_name: str, Name of resource, default=""

Reply:
Fiksa